packages/auto-id/package.json

```diff
@@ -10,7 +10,10 @@
   },
   "dependencies": {
     "@autonomys/auto-utils": "workspace:*",
-    "@types/node": "^20.12.12"
+    "@peculiar/asn1-schema": "^2.3.8",
+    "@peculiar/asn1-x509": "^2.3.8",
+    "@types/node": "^20.12.12",
```

Comment:
Dependencies that only include types (so anything `@types/*`) should be in the `devDependencies` since they are only required at compilation time.